Cannot find type definition file for 'mocha'

When installing f-streams-async in a project that does not have @types/mocha, we have the following typescript error

```
node_modules/f-streams-async/build/lib/node-wrappers.d.ts:5:23 - error TS2688: Cannot find type definition file for 'mocha'.

5 /// <reference types="mocha" />
                        ~~~~~

Found 1 error in node_modules/f-streams-async/build/lib/node-wrappers.d.ts:5
```

This included reference to typescript seem to be due to `NodeJS.EventEmitter` used in `lib/node-wrappers.ts`. This type exists in both `node` and `mocha`.

@tchambard Do you have an idea ?

--- a/lib/node-wrappers.ts
+++ b/lib/node-wrappers.ts
@@ -23,6 +23,7 @@ import * as https from 'https';
 import * as net from 'net';
 import * as os from 'os';
 import * as stream from 'stream';
+import type { EventEmitter } from 'stream';
 import { parse as parseUrl } from 'url';
 import * as generic from './devices/generic';
 import { Reader } from './reader';
@@ -37,7 +38,7 @@ import { Writer } from './writer';
 /// * `wrapper = new streams.Wrapper(stream)`
 ///   creates a wrapper.
 
-export interface Emitter extends NodeJS.EventEmitter {
+export interface Emitter extends EventEmitter {
     end?: (data?: any, encoding?: string) => void;
     close?: () => void;
     destroySoon?: () => void;

